src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -85,6 +85,7 @@ def get(self, request: Request, project) -> Response:
             if i.has_feature(IntegrationFeatures.STACKTRACE_LINK)
         ]
 
+        last_error = None
```

Comment:
We're still going to report back the last_error even though I believe it makes no sense when there's more than one code mapping.